patient.go

```diff
@@ -0,0 +1,107 @@
+package client
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/Joshswooft/nhs/cmd/validation"
+	"github.com/welldigital/nhs-fhir/model"
+)
+
+type PatientService = service
+
+const path = "Patient"
+
+// Get gets a patient from the PDS using the patients NHS number as the id.
+// id = The patient's NHS number. The primary identifier of a patient, unique within NHS England and Wales. Always 10 digits and must be a valid NHS number.
+func (p *PatientService) Get(ctx context.Context, id string) (*model.Patient, *Response, error) {
```

Comment:
do we need to return the original http response?

Reply:
Hey, yep this will be needed for the following reasons:

1. Request ID - Every request to the API has to have a unique request ID. So this is useful if the user ever needs to go the owner of the API...
2. Observability 
3. Can be extended with things like pagination, count, token expiration time, ...
4. Most Clients in various programming languages do this so considering it a standard.

Reply:
As needed by this: https://github.com/welldigital/NHS-FHIR/pull/8